src/main/java/com/iwaconsolti/league/service/LeagueService.java

```diff
@@ -0,0 +1,95 @@
+package com.iwaconsolti.league.service;
+
+import com.iwaconsolti.league.Config.LeagueConfig;
+import com.iwaconsolti.league.model.LeagueModel;
+import com.iwaconsolti.league.model.TeamModel;
+import com.iwaconsolti.league.model.PlayerModel;
+import com.iwaconsolti.league.model.MatchModel;
+import com.iwaconsolti.league.Config.MatchConfig;
+import com.iwaconsolti.league.Config.TeamConfig;
+import com.iwaconsolti.league.Config.PlayerConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Profile("populated")
+public class LeagueService {
+
+    private final LeagueModel soccerLeague;      //Bean
+    private final LeagueModel basketballLeague;  //Bean
+
+    private final MatchConfig matchConfig;
+    private final TeamConfig teamConfig;
+    private final PlayerConfig playerConfig;
+    private final LeagueConfig leagueConfig;
+
+    @Autowired
+    public LeagueService(
+            @Qualifier("soccer") LeagueModel soccerLeague,
+            @Qualifier("Basketball") LeagueModel basketballLeague,
```

Comment:
Injects dependencies; consider renaming 'Basketball' qualifier to lowercase for uniformity.